### PR TITLE
Adds upcoming events to /meet-the-team

### PR DESCRIPTION
**Pull request by johnrich85**
Changes proposed in this pull request:

Adds upcoming events to /meet-the-team

### CODE REVIEW

File: site/en/meet-the-team/events/city-js-london-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: City JS London 2023
+summary: >-
+  A JavaScript festival focused on JS frameworks, NodeJS,
+  TypeScript, Testing, and more.
```
Comment (rachelandrew) [inline site/en/meet-the-team/events/city-js-london-2023/index.md:5]:
```suggestion
  A JavaScript festival focused on JavaScript frameworks, NodeJS,
  TypeScript, testing, and more.
```

File: site/en/meet-the-team/events/city-js-london-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - speaker: jecelynyeen
+    title: How browser automation works behind the scenes?
+    description: >-
+      If you have ever worked with browser automation, you know how overwhelming it can
+      be. Why is browser automation hard? How can the browser vendors improve the
+      fundamentals of browser automation?
+
+      In this talk, Jecelyn will walk you through the current state of cross-browser
+      automation, unfold how browser automation works behind the scenes, and share
+      about the motivation of WebDriver BiDi – the upcoming browser automation protocol.
```
Comment (rachelandrew) [inline site/en/meet-the-team/events/city-js-london-2023/index.md:20]:
```suggestion
      about the motivation of WebDriver BiDi–the upcoming browser automation protocol.
```

File: site/en/meet-the-team/events/selenium-conf-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Selenium Conference
+summary: >-
+  SeleniumConf brings together Selenium and WebDriver ecosystem developers &
```
Comment (rachelandrew) [inline site/en/meet-the-team/events/selenium-conf-2023/index.md:4]:
```suggestion
  SeleniumConf brings together Selenium and WebDriver ecosystem developers and
```

File: site/en/meet-the-team/events/smashingconf-freiburg-2023/index.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+title: SmashingConf Freiburg 2023
+summary: >-
+  The 11th SmashingConf Freiburg - all about front-end, UX and design - for
+  front-end developers and designers. With adventures into design systems,
+  accessibility, CSS, JavaScript, web performance and cutting-edge techniques.
```
Comment (rachelandrew) [inline site/en/meet-the-team/events/smashingconf-freiburg-2023/index.md:6]:
```suggestion
  The 11th SmashingConf Freiburg—all about front-end, UX and design—for
  front-end developers and designers. With adventures into design systems,
  accessibility, CSS, JavaScript, web performance and cutting-edge techniques.
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Just a bit of copyediting, also I know that An Event Apart is not happening as the event series has closed down. So we should remove this one.

**Comment by Bryan679** [comment]
Here are some times that work for me:


Friday, February 10, 2023



1:00 AM - 1:30 AM (GMT-8)



9:30 AM - 10:00 AM (GMT-8)




Made easy by Outlook<https://aka.ms/Dcupce>




Sent from my Boost Samsung Galaxy A03s
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**